### DISCUSSION TITLE
Can I adjust `max_slot_wal_keep_size`?

### DISCUSSION THREAD

**Discussion by Tang8330** [supabase/supabase#27457] [Questions]
It seems like the default setting is 1 GB and from the docs [here](https://supabase.com/docs/guides/platform/custom-postgres-config) - it does not seem like it's a configurable setting?

Wanted to double check:
1. Is this configurable? If so, how?
2. If this is not configurable, why is that?

**Comment by GaryAustin1** [comment]
Seems like it is going to 4GB.  And per that link it is not configurable.  
https://github.com/supabase/postgres/pull/981
